lnst/RecipeCommon/Perf/Measurements/Results/XDPBenchMeasurementResults.py

```diff
@@ -1,20 +1,43 @@
 from lnst.RecipeCommon.Perf.Results import ParallelPerfResult
-from lnst.RecipeCommon.Perf.Measurements.Results.FlowMeasurementResults import (
-    FlowMeasurementResults,
+from lnst.RecipeCommon.Perf.Measurements.Results.BaseMeasurementResults import (
+    BaseMeasurementResults,
 )
 from lnst.RecipeCommon.Perf.Measurements.MeasurementError import MeasurementError
 
 
-class XDPBenchMeasurementResults(FlowMeasurementResults):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+class XDPBenchMeasurementResults(BaseMeasurementResults):
```

Comment:
Can't we keep inheriting from FlowMeasurementResults and only override problematic properties that seem to be `start_timestamp` and `end_timestamp`?

Reply:
Ok, resolving.